### PR TITLE
Speeding the tests for whisper

### PR DESCRIPTION
**Pull request by susnato**
What does this PR do?
This PR modifies tests as described in https://github.com/keras-team/keras-nlp/issues/866 for `whisper`.

**Results**
before - 9 passed, 3 skipped in 22.53s
after - 8 passed, 4 skipped in 7.96s
**2.83 times speedup(on my local machine)**

@mattdangerw @chenmoneygithub

### CODE REVIEW

File: keras_nlp/models/whisper/whisper_backbone_test.py [hunk 1]
```diff
@@ -143,7 +128,7 @@ def test_saved_model(self, save_format, filename):
 class WhisperBackboneTPUTest(tf.test.TestCase, parameterized.TestCase):
     def setUp(self):
         with self.tpu_strategy.scope():
-            self.model = WhisperBackbone(
+            self.backbone = WhisperBackbone(
                 vocabulary_size=1000,
```
Comment (mattdangerw) [inline keras_nlp/models/whisper/whisper_backbone_test.py:132]: let's size down this backbone same as above.

**Review by mattdangerw — CHANGES_REQUESTED** [review]
Looks good! Just one comment

**Comment by susnato** [comment]
Hi @mattdangerw I made those changes that you requested. Please check

**Comment by mattdangerw** [comment]
LGTM!

**Review by mattdangerw — APPROVED** [review]

**Pull request merged**